CHANGELOG.md

```diff
@@ -0,0 +1,22 @@
+# Changelog
+
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## [Unreleased]
+### Added
+- This changelog file ([#273])
+
+
+## [2.4.0] - 2018-08-08
+### Deprecated
+
+* Support for Python 2.7 ([#265])
+
+
+[Unreleased]: https://github.com/olivierlacan/keep-a-changelog/compare/v2.4.0...HEAD
```

Comment:
Forgot to update this line. Will be fixed by https://github.com/pylast/pylast/pull/282.